tests/e2e/scenarios/test_admin_api.py

```
@@ -142,6 +142,86 @@ async def test_get_user_detail(admin_client, test_user):
+    await expect(
+        detail_header.get_by_text(
+            SEL_V2["admin_member_role_name"], exact=True
+        )
+    ).to_be_visible()
+    await expect(
+        detail_header.get_by_text(
+            SEL_V2["admin_active_status_name"], exact=True
+        )
+    ).to_be_visible()
+
+    async def set_role(role_name):
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `set_role`

Add return type annotation: `None`

(ANN202)